src/Hadrian/Haskell/Cabal/Parse.hs-boot

```diff
@@ -0,0 +1,9 @@
+module Hadrian.Haskell.Cabal.Parse where
```

Comment:
Is there a chance to get rid of this cyclic import by adding another `.Type` module somewhere?

Reply:
@angerman thoughts? I'm suspecting "yes".